raiden/transfer/node.py

```diff
@@ -592,7 +592,7 @@ def handle_new_token_network(
     return TransitionResult(chain_state, events)
 
 
-def handle_node_change_network_state(
+def handle_action_node_change_network_state(
```

Comment:
same here: What's the exact naming convention? The state change is `ActionChangeNodeNetworkState` so I would expect `handle_action_change_node_network_state`

Reply:
Amended the changes.